<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>560</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>560</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CURT Bookstore</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(237, 28, 36);
border-radius: 5px;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0" columnstretch="0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabBar::tab{color:white; background:#ed1c24; border:.5px solid white; border-bottom-color:#ed1c24;     border-top-left-radius:10px; border-top-right-radius:10px; padding:8px 100px; margin-bottom:-2px;}
QTabBar::tab:selected {background:white;color:#ed1c24; border-color:white; border-bottom-color:#ed1c24;}
QTabWidget::pane { border: .5px solid white; }
QLabel {color:white;}
QLineEdit{background-color:white;margin-bottom:5px;padding:5px;border-radius:12px}
QPushButton {color:white;
border: 1px solid white;
border-radius: 12px;}
QPushButton::hover {color:#ed1c24; background-color:white}</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="lib_tab">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <attribute name="title">
       <string>Library</string>
      </attribute>
      <widget class="QListWidget" name="avBooks_list">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>471</width>
         <height>321</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid white;padding:5px;
color:white;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>-5</y>
         <width>461</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans MT</family>
         <pointsize>17</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Books</string>
       </property>
       <property name="margin">
        <number>7</number>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>71</y>
         <width>1</width>
         <height>282</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>551</x>
         <y>202</y>
         <width>493</width>
         <height>1</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>60</y>
         <width>441</width>
         <height>158</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="bookTitleLabel">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Book Title:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="bookTitleLineEdit">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="authorLabel">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="authorLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="bookTypeLabel">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Book Type:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="bookTypeLineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>-5</y>
         <width>461</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans MT</family>
         <pointsize>17</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Search by</string>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>210</y>
         <width>461</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans MT</family>
         <pointsize>17</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Selected Book Info</string>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>260</y>
         <width>381</width>
         <height>94</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Book Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Author:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Section (Book Type):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Cost:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="inf_name_label">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>N/a</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="inf_author_label">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>N/a</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="inf_section_label">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>N/a</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="inf_cost_label">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0$</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="buyBook_pButton">
       <property name="geometry">
        <rect>
         <x>880</x>
         <y>320</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans MT</family>
         <pointsize>10</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Buy Book</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>675</x>
         <y>17</y>
         <width>371</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>7</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgba(245, 245, 245, 150);</string>
       </property>
       <property name="text">
        <string>(You can search by one, two or multiple parameters, just type in!)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_done">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>360</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(68, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_failed">
       <property name="geometry">
        <rect>
         <x>760</x>
         <y>360</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0);background-color:rgba(0,0,0,0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label</zorder>
      <zorder>avBooks_list</zorder>
      <zorder>line</zorder>
      <zorder>formLayoutWidget</zorder>
      <zorder>line_2</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>horizontalLayoutWidget</zorder>
      <zorder>buyBook_pButton</zorder>
      <zorder>label_16</zorder>
      <zorder>label_done</zorder>
      <zorder>label_failed</zorder>
     </widget>
     <widget class="QWidget" name="profit_tab">
      <attribute name="title">
       <string>Profit</string>
      </attribute>
      <widget class="QListWidget" name="soldBooks_list">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>471</width>
         <height>321</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Semibold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid white;padding:5px;
color:white;</string>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>71</y>
         <width>1</width>
         <height>282</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>-5</y>
         <width>461</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans MT</family>
         <pointsize>17</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Sold Books</string>
       </property>
       <property name="margin">
        <number>7</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>40</y>
         <width>461</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans MT</family>
         <pointsize>17</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Summary</string>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>100</y>
         <width>381</width>
         <height>94</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="tBooks_sold_label">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="tProfit_label">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>0$</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Total Books Sold:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Total Profit:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="returnBook_pButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>210</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans MT</family>
         <pointsize>10</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Return Book</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_failed_2">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>260</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0);background-color:rgba(0,0,0,0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_done_2">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>260</y>
         <width>291</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(68, 255, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>label_done_2</zorder>
      <zorder>label_12</zorder>
      <zorder>soldBooks_list</zorder>
      <zorder>line_3</zorder>
      <zorder>label_13</zorder>
      <zorder>horizontalLayoutWidget_2</zorder>
      <zorder>returnBook_pButton</zorder>
      <zorder>label_failed_2</zorder>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="curt_logo">
     <property name="minimumSize">
      <size>
       <width>253</width>
       <height>110</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>253</width>
       <height>110</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>src/transparents2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
